app/views/admin/categories/_form.html.haml

```diff
@@ -23,21 +23,27 @@
           = f.text_field :acronym, class: "form-control", readonly: true
     %tr
       %th
-        Name:
+        Name
         %span.asterik *
       %td.top
         = f.text_field :name, class: "form-control"
     %tr
       %th
-        Description:
+        Description
       %td.top
         = f.text_area :description, class: "form-control"
     - unless new_record
       %tr
         %th
-          Created:
+          Created
         %td.top
           = f.text_field :created, readonly: true, class: "form-control"
+      %tr
+        %th
+          Ontologies
+        %td.top
+          = render SelectInputComponent.new(id: @id, name: @name, values: @values , selected: @selected , multiple: @multiple, hide: true)
```

Comment:
why hide it?

Reply:
we just need the select 
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/31127782/c0436f90-ba5d-49f0-819c-c063f252fb84)

Reply:
OK, you need to make the variable name more clear, from `hide` to 'open_to_add_values' or something like this

Reply:
Done